clang/docs/ReleaseNotes.rst

```diff
@@ -678,6 +678,12 @@ Improvements to Clang's diagnostics
 - Clang now accepts ``@tparam`` comments on variable template partial
   specializations. (#GH144775)
 
+- Fixed a bug that caused diagnostic line wrapping to not function correctly on
+  some systems. (#GH139499)
+
+- Clang now tries to avoid printing file paths that contain ``..``, instead preferring
```

Comment:
We also handle things like `.` don't we? I suppose that doesn't matter for the release note.